common/app/views/fragments/nav/newHeaderMenu.scala.html

```diff
@@ -4,36 +4,35 @@
 @import conf.switches.Switches.SearchSwitch
 
 @sectionList(topLevelSection: NewNavigation.EditionalisedNavigationSection) = {
-    <li class="menu-item js-navigation-item"
-        data-section-name="@topLevelSection.name">
-        <details class="js-close-nav-list"
-                 id="primary-list-@topLevelSection.name"
-                 open>
-            <summary class="menu-item__title js-navigation-button"
-                     data-link-name="nav2 : secondary : @topLevelSection.name">
+    @defining(Edition(request)) { edition =>
+        <li class="menu-item js-navigation-item"
+            data-section-name="@topLevelSection.name">
+            <a href="@{topLevelSection.getPopularEditionalisedNavLinks(edition)(0).url}"
```

Comment:
Actually @paperboyo you are right! I had forgotten, but thats actually how @zeftilldeath had designed it. I like that option the best.